### ISSUE TITLE
some question about lr

### ISSUE THREAD

**Issue by LYMDLUT** [Atten4Vis/ConditionalDETR#27]
Non-DC5 models were trained with batch size 2, and DC5 with 1，should the learning rate changed？

**Comment by charlesCXK** [comment]
Hi, we didn't change learning rate.
